scripts/lint-style.lean

```diff
@@ -80,13 +82,13 @@ def checkInitImports : IO Bool := do
   if missing.size > 0 then
     IO.eprintln s!"error: the following {missing.size} module(s) do not import Mathlib.Init: \
       {missing}"
-    return true
-  return false
+    return missing.size
+  return 0
 
 
 /-- Verifies that every file in the `scripts` directory is documented in `scripts/README.md`.
-Return `True` if there are undocumented scripts, otherwise `False`. -/
-def allScriptsDocumented : IO Bool := do
+Return the number of undocumented scripts (if any). -/
```

Comment:
```suggestion
Return the number of undocumented scripts. -/
```
I would say that returning `0` is returning the number of undocumented scripts!  If you want to be more explicit about it, you can say something like "returning `0` if all scripts are documented".